addon/styles/_frost-expand.scss

```diff
@@ -0,0 +1,56 @@
+$frost-color-expand-label-text: $frost-color-blue-1 !global;
+$frost-color-expand-scroll-border: $frost-color-lgrey-3 !global;
+$frost-color-expand-content-bg: $frost-color-lgrey-4 !global;
+$frost-color-expand-content-text: $frost-color-grey-3 !global;
+
+.frost-expand {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  font-family: $frost-font-family;
+
+  &.collapsed {
+    .frost-expand-label-icon {
+      transform: rotate(-90deg);
+    }
+  }
+
+  &.expanded {
+    .frost-expand-label-icon {
+      transform: rotate(0deg);
+    }
+  }
+
+  &-label {
+    display: flex;
+    flex-direction: row;
+    flex-shrink: 0;
+    justify-content: flex-start;
```

Comment:
Purpose was to avoid this: 
![screen shot 2017-05-26 at 11 30 50 am](https://cloud.githubusercontent.com/assets/22083831/26501184/d0ac77f2-4206-11e7-8baf-365e51e05f39.png)

is there a better way?